src/pages/Signup.jsx

```diff
@@ -1,5 +1,47 @@
+import { Link } from "react-router-dom";
 function Signup() {
-  return <h1>Signup</h1>;
```

Comment:
Are you yet to do functionality here?

Reply:
Currently, Signup functionality is not in action.
Auth only works for the users already created in the db.